bundle-workflow/src/test_workflow/integ_test/local_test_cluster.py

```diff
@@ -55,21 +60,31 @@ def destroy(self):
             logging.info("Local test cluster is not started")
             return
         self.terminate_process()
+        logs_files = walk(os.path.join(self.work_dir, self.install_dir, "logs"))
+        self.test_recorder.record_local_cluster_logs(self.component_name, self.component_config, self.local_cluster_stdout,
```

Comment:
For clarity, put every argument on its own line

```python
self.test_recorder.record_local_cluster_logs(
     self.component_name, 
```

Reply:
Done